operations/loyaltymemberships.md

```diff
@@ -88,7 +89,8 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
             "IsPrimary": false,
             "Points": 25,
             "ExpirationDate": "",
-            "Url": ""
+            "Url": "",
+            "LoyaltyTierId": ""
```

Comment:
```suggestion
            "LoyaltyTierId": null
```

Since this is a nullable `Guid` it should be like that.

PS. I would say the same logic applies to `ExpirationDate` but thats just nit pick, not related to this PR, its up to you.